conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
Ok, I see, it makes sense as a limitation of argparse (actually a limitation of the command line parsing approach), if the positional argument is not specified before the optional one, it gets messed. So I would just assume that the argument would be ```conan info .. --graph filename.dot``, always using first the positional one.

Or just leave it as is, not big issue, if users want to have different output filenames, they will requested, and can be added later. Up to you.